Sources/_InternalTestSupport/PlatformHelpers.swift

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Basics
+
+public func isMacOs() -> Bool {
+    #if os(macOS)
+    return true
+    #else
+    return false
+    #endif
+}
+
+public func isRealSigningIdentityTestEnabled() -> Bool {
+    #if ENABLE_REAL_SIGNING_IDENTITY_TEST
+    return true
+    #else
+    return false
+    #endif
+}
+
+public func isEnvironmentVariableSet(_ variableName: EnvironmentKey) -> Bool {
+    guard let value = Environment.current[variableName] else {return false}
```

Comment:
nit: run the formatter

Reply:
Although I understand the desire to have consistent formatting, it is not currently enforced via the pipelines.  Having the tools enforce "formatting" and "linting" is the correct solution, in my opinion.  Until such time, I will do my best to run the formatter, however it will cause significant changes for the sources we touch as not all lines of code adhere to the formatter settings.